src/main/java/net/sf/jabref/sql/exporter/DBExporter.java

```diff
@@ -131,10 +131,12 @@ private void populateEntriesTable(int database_id,
                 query = query + ", ";
                 val = entry.getField(SQLUtil.getAllFields().get(i));
                 if (val != null) {
-                    val = val.replace("\\", "\\\\");
-                    val = val.replace("\"", "\\\"");
-                    val = val.replace("\'", "''");
-                    val = val.replace("`", "\\`");
+                    if(dbStrings.getServerType().equals("MySQL")){
```

Comment:
I would love to see a comment here why this is necessary. So future developers can instantly see the reason behind this code and remove it if behaviour has changed.